### PR TITLE
#171018283 Chore sqlite testing

### PR DESCRIPTION
**Pull request by Frediflexta**
#### What does this PR do?
- it creates a sqlite database for testing
#### Description of Task to be completed?
- config database.php file to point to the right file
- add code coverage to our tests
- config composer.json with a script to run tests subsequently
#### How should this be manually tested?
- please refer to the readme to setup your machine with the project
- write your test and run `composer test` 
- or test a file by running `composer test-one <Name-of-Test-file> `
#### Any background context you want to provide?
- without this PR, the tests will need to create its own database for test, and as the project scales, running test will take alot of time to run 
#### What are the relevant pivotal tracker stories?
- [#171018283](https://www.pivotaltracker.com/story/show/171018283)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -24,6 +24,11 @@
...
         </whitelist>
     </filter>
+    <logging>
+        <log type="coverage-html" target="./report" lowUpperBound="35" highLowerBound="70"/>
+        <log type="coverage-clover" target="./coverage.xml"/>
+    </logging>
+
     <php>
         <server name="APP_ENV" value="testing"/>
         <server name="BCRYPT_ROUNDS" value="4"/>
```
Comment (anasey01) [inline phpunit.xml]: We're using BCRYPT_ROUNDS of 12

File: composer.json [hunk 2]
```diff
@@ -63,6 +63,13 @@
...
+        ],
+        "test": [
+            "vendor/bin/phpunit --debug --verbose --coverage-clover build/logs/clover.xml"
+        ],
+        "test-one": [
+            "vendor/bin/phpunit --filter"
         ]
+
     }
 }
```
Comment (anasey01) [inline composer.json]: Please add a new line here

File: .gitignore [hunk 3]
```diff
@@ -11,3 +11,6 @@ Homestead.yaml
 npm-debug.log
 yarn-error.log
 .DS_Store
+
+/report
+build/
```
Comment (anasey01) [inline .gitignore]: Please add a new line here

File: phpunit.xml (hunk 1 above)
Comment (Frediflexta) [reply phpunit.xml]: That value is there by default, and doesn't have anything to do with the testing db I just set up.

**Pull request merged**